tests/src/test/java/alluxio/client/cli/fs/command/GetCmdStatusCommandTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.cli.fs.command;
+
+import alluxio.cli.fs.FileSystemShell;
+import alluxio.client.cli.fs.AbstractFileSystemShellTest;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.FileSystemTestUtils;
+import alluxio.conf.PropertyKey;
+import alluxio.conf.ServerConfiguration;
+import alluxio.exception.AlluxioException;
+import alluxio.grpc.WritePType;
+import alluxio.master.LocalAlluxioJobCluster;
+import alluxio.testutils.LocalAlluxioClusterResource;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.IOException;
+
+/**
+ * Tests for getting cmd status {@link alluxio.cli.fs.command.GetCmdStatusCommand}.
+ */
+public class GetCmdStatusCommandTest extends AbstractFileSystemShellTest  {
```

Comment:
Is there a test case where `--wait` is false??

Reply:
no, tested manually in cli, i will add one in unit test.